libc/include/assert.h.def

```diff
@@ -12,4 +12,21 @@
 // This file may be usefully included multiple times to change assert()'s
 // definition based on NDEBUG.
 
+#ifndef __cplusplus
+#undef static_assert
+#define static_assert _Static_assert
+#endif
+
+#undef assert
+#ifdef NDEBUG
+#define assert(e) (void)0
+#else
+#ifdef __cplusplus
+extern "C"
+#endif
```

Comment:
This file already includes `__llvm-libc-common.h` so you can use `__BEGIN_C_DECLS` and `__END_C_DECLS`.